stripe/src/main/java/com/stripe/android/StripeApiBeta.kt

```diff
@@ -0,0 +1,10 @@
+package com.stripe.android
+
+/**
+ * Enums of beta headers allowed to be override when initializing [Stripe].
+ */
+enum class StripeApiBeta(val code: String) {
+    @Deprecated("alipay is public")
+    AlipayV1("alipay_beta=v1"),
```

Comment:
I guess it helps us test what happens when the set has more than one element. We should probably just omit this - it no longer is relevant.

Reply:
done